en/docs/tutorials/streaming-api/create-and-publish-websub-api.md

```diff
@@ -0,0 +1,257 @@
+# Create and Publish a WebSub API
+
+Follow the instructions in this tutorial to design and publish a WebSub API, and register a webhook to it.
+
+The tutorial demonstrates a simple WebSub API that monitors your GitHub repository for new issues, and receives events when an issue is created.
+
+<html>
+<div class="admonition note">
+<p class="admonition-title">Note</p>
+<p>For more information on WebSub APIs, see <a href="{{base_path}}/design/create-api/create-streaming-api/create-a-websub-streaming-api">Create a WebSub API</a>.</p>
+</div> 
+</html>
+
+### Step 1 - Design a WebSub API
+
+1.  Sign in to the WSO2 API Publisher `https://<hostname>:9443/publisher` (e.g., `https://localhost:9443/publisher`).
+
+2.  Go to **CREATE API** and Click **Design New Streaming API**.
+
+    [![Design New Streaming API]({{base_path}}/assets/img/learn/design-api/streaming-api/design-new-streaming-api.png)]({{base_path}}/assets/img/learn/design-api/streaming-api/design-new-streaming-api.png)
+
+    <html><div class="admonition note">
+      <p class="admonition-title">Note</p>
+      <p>The <b>CREATE</b> button will only appear for a user who has the <code>creator</code> role permission.</p>
```

Comment:
Similar to [1].
[1] https://github.com/wso2/docs-apim/pull/3284/files#r589208308